zone/client_packet.cpp

```diff
@@ -16392,3 +16395,92 @@ void Client::Handle_OP_RaidClearNPCMarks(const EQApplicationPacket* app)
 		r->RaidClearNPCMarks(this);
 	}
 }
+
+void Client::RecordStats()
+{
+	auto r = CharacterStatsRecordRepository::FindOne(
+		database,
+		CharacterID()
+	);
+
+	r.status                   = Admin();
+	r.name                     = GetCleanName();
+	r.aa_points                = GetAAPoints() + GetSpentAA();
+	r.level                    = GetLevel();
+	r.class_                   = GetBaseClass();
+	r.race                     = GetBaseRace();
+	r.aa_points                = GetAAPoints();
```

Comment:
Did the unspent part get removed?

Reply:
Oh nope, just duplicate information.

Reply:
Currently aa_points are unspent aa points, should have both, spent and unspent, possibly as separate fields?

Reply:
They're as both a few lines above, removed the duplicate